public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -577,7 +577,7 @@ export const Explorer = ({
                       explorerFields={explorerFields}
                       timeStampField={queryRef.current![SELECTED_TIMESTAMP]}
                       rawQuery={appBasedRef.current || queryRef.current![RAW_QUERY]}
-                      totalHits={explorerData?.datarows?.length || 0}
+                      totalHits={_.sum(countDistribution.data['count()'])}
```

Comment:
It is ok for now but ideally we should change the parameter where it gets count distribution data, originally it is `xxx/_ppl?format=viz, but now should be xxx/_ppl?format=jdbc`

Reply:
Will include in your later look and feel PR